streaming/options.go

```diff
@@ -18,3 +22,13 @@ func WithConnectCallback(fn ConnectCallback) DialOption {
 		c.connectCallback = fn
 	}
 }
+
+// WithBackoffHandler specify custom handler to calculate backoff duration after each disconnect. Attempt will increment
```

Comment:
Minor adjustment if we consider `WithBackoffHandler` as part of the sentence 👍 
```suggestion
// WithBackoffHandler specifies a custom handler to calculate backoff duration after each disconnect. Attempt will increment
```